tutorials/tour/basics.md

```diff
@@ -37,51 +37,52 @@ println("Hello," + " world!") // Hello, world!
 
 ### Values
 
-You can name results of expressions by using the `val` keyword.
+You can name results of expressions with the `val` keyword.
 
-```
+```tut
 val x = 1 + 1
-println(x) // 2
+println(x * x) // 4
```

Comment:
Personally, it looks like it's adding unnecessary complication only to show that values hold results of expressions.

What's the reason for this change?